docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
     ports:
       - 4444:4444
     volumes:
-      - $HOME/.msf4:/root/.msf4
+      - /tmp/.msf4:/home/msf/.msf4
```

Comment:
@ilyaglow I did it like this before in my Dockerfiles:
```
RUN mkdir -p dir
RUN chown -R user:user dir
VOLUME ["dir"]
```

This way the folder will be owned by the user in the docker image. Can you test if this also works here?